espnet2/st/espnet_model.py

```diff
@@ -161,8 +161,8 @@ def forward(
         speech_lengths: torch.Tensor,
         text: torch.Tensor,
         text_lengths: torch.Tensor,
-        src_text: Optional[torch.Tensor],
-        src_text_lengths: Optional[torch.Tensor],
+        src_text: Optional[torch.Tensor] = None,
+        src_text_lengths: Optional[torch.Tensor] = None,
```

Comment:
No need to set it to None as it is already Optional
```suggestion
        src_text: Optional[torch.Tensor],
        src_text_lengths: Optional[torch.Tensor],
```